code/gradle/release.gradle

```diff
@@ -248,27 +351,27 @@ task checksum(dependsOn: assembleArtifacts) << {
 } 
 
 task prepareRelease (dependsOn: ['clean', 'build', 'genDataList']) {
-	description="Prepare PCGen for release. This will update and commit the version, do a clean build including the unit tests and generate the data list for the installer."
+    description="Prepare PCGen for release. This will update and commit the version, do a clean build including the unit tests and generate the data list for the installer."
 }
 
 task pcgenRelease (dependsOn: ['build', 'assembleArtifacts', 'checksum', 'tagRelease']) {
-	description="Release a new version of PCGen. This will build PCGen, produce the release artifacts, tag the release and then update and commit the version ready for new dev work."
+    description="Release a new version of PCGen. This will build PCGen, produce the release artifacts, tag the release and then update and commit the version ready for new dev work."
 }
 
 // Update the PCGen version to indicate a release, then commit the new version
 task updateVersionRelease << {
-	unSnapshotVersion()
-	commitFile(new File("gradle.properties"), "Update version for release ${project.version}")
+    unSnapshotVersion()
+    commitFile(new File("gradle.properties"), "Update version for release ${project.version}")
```

Comment:
I'm wondering if this actually does anything. When originally written, this was an SVN based design (tagging the release actually functioned). I'm not sure this is functioning now that we use git and github.com.